landing-page/content/common/spark-quickstart.md

```diff
@@ -0,0 +1,325 @@
+
+{{% codetabs "SelectData" %}}
+{{% addtab "SparkSQL" "spark-queries" "spark-sql" %}}
+{{% addtab "Spark-Shell" "spark-queries" "spark-shell" %}}
+{{% addtab "PySpark" "spark-queries" "pyspark" %}}
+{{% tabcontent "spark-sql"  %}}
+```sql
+SELECT * FROM demo.nyc.taxis;
+```
+{{% /tabcontent %}}
+{{% tabcontent "spark-shell" %}}
+```scala
+val df = spark.table("demo.nyc.taxis").show()
+```
+{{% /tabcontent %}}
+{{% tabcontent "pyspark" %}}
+```py
+df = spark.table("demo.nyc.taxis").show()
+```
+{{% /tabcontent %}}
+{{% /codetabs %}}
+
+
+### Adding A Catalog
+
+Iceberg has several catalog back-ends that can be used to track tables, like JDBC, Hive MetaStore and Glue.
+Catalogs are configured using properties under `spark.sql.catalog.(catalog_name)`. In this guide,
+we use JDBC, but you can follow these instructions to configure other catalog types. To learn more, check out
+the [Catalog](../docs/latest/spark-configuration/#catalogs) page in the Spark section.
+
+This configuration creates a path-based catalog named `local` for tables under `$PWD/warehouse` and adds support for Iceberg tables to Spark's built-in catalog.
+
+
+{{% codetabs "AddingACatalog" %}}
+{{% addtab "CLI" "spark-init" "cli" %}}
+{{% addtab "spark-defaults.conf" "spark-init" "spark-defaults" %}}
+{{% tabcontent "cli"  %}}
+```sh
+spark-sql --packages org.apache.iceberg:iceberg-spark-runtime-3.2_2.12:{{% icebergVersion %}}\
+    --conf spark.sql.extensions=org.apache.iceberg.spark.extensions.IcebergSparkSessionExtensions \
+    --conf spark.sql.catalog.spark_catalog=org.apache.iceberg.spark.SparkSessionCatalog \
+    --conf spark.sql.catalog.spark_catalog.type=hive \
+    --conf spark.sql.catalog.demo=org.apache.iceberg.spark.SparkCatalog \
+    --conf spark.sql.catalog.demo.type=hadoop \
+    --conf spark.sql.catalog.demo.warehouse=$PWD/warehouse \
+    --conf spark.sql.defaultCatalog=demo
+```
+{{% /tabcontent %}}
+{{% tabcontent "spark-defaults" %}}
+```sh
+spark.jars.packages                                  org.apache.iceberg:iceberg-spark-runtime-3.2_2.12:{{% icebergVersion %}}
+spark.sql.extensions                                 org.apache.iceberg.spark.extensions.IcebergSparkSessionExtensions
+spark.sql.catalog.spark_catalog                      org.apache.iceberg.spark.SparkSessionCatalog
+spark.sql.catalog.spark_catalog.type                 hive
+spark.sql.catalog.demo                               org.apache.iceberg.spark.SparkCatalog
+spark.sql.catalog.demo.type                          hadoop
+spark.sql.catalog.demo.warehouse                     $PWD/warehouse
+spark.sql.defaultCatalog                             demo
+```
+{{% /tabcontent %}}
+{{% /codetabs %}}
+
+
+{{< hint info >}}
+If your Iceberg catalog is not set as the default catalog, you will have to switch to it by executing `USE demo;`
+{{< /hint >}}
+
+### Next steps
+
+#### Adding Iceberg to Spark
+
+If you already have a Spark environment, you can add Iceberg, using the `--packages` option.
```

Comment:
For a real deployment, you'd want to add the runtime Jar to your Spark install's `jars` folder. I think this should say that first (and link to the 3.2 Jar) and then add that you can add Iceberg to a single Spark session using the `--packages` option.

Also, this should point the reader to catalogs, since that's how you would add a catalog.